.changeset/lucky-maps-confess.md

```diff
@@ -0,0 +1,5 @@
+---
+'@roadiehq/rag-ai-backend-retrieval-augmenter': patch
```

Comment:
Let's make this major since we are changing interfaces. That way we can get to the mainline versioning strategy at the same time.

Reply:
updated!